sdk/src/types/block/payload/signed_transaction/transaction.rs

```diff
@@ -328,16 +328,14 @@ impl WorkScore for Transaction {
     }
 }
 
-fn verify_network_id<const VERIFY: bool>(network_id: &u64, visitor: &ProtocolParameters) -> Result<(), Error> {
-    if VERIFY {
```

Comment:
Just checking, visitor doesn't need to be made optional and then a check if it's Some()?